lib/assessments/permission.rb

```diff
@@ -73,23 +77,23 @@ def revoke_level(user)
           assessment.viewers.destroy(user)
         when :network_partner
           assessment.network_partners.destroy(user)
-        assessment.reload
+          assessment.reload
```

Comment:
I originally thought that, but it seems that the original statement had it encapsulated inside of the last `when` block.  I'm going to triple check that this is the case; RubyMine can be pretty zealous about code formatting.